### ISSUE TITLE
Change usage of CURL to ansible uri module 

### ISSUE THREAD

**Issue by tayzlor** [Capgemini/kubeform#28]
we need to wait on Ansible 2.1 (so the dependency on httplib is removed for CoreOS) - 

this code needs updated - https://github.com/Capgemini/kubeform/blob/master/roles/k8s-master/tasks/main.yml#L88

and we need to modify wercker.yml

http://docs.ansible.com/ansible/uri_module.html

**Comment by tayzlor** [comment]
Closing - no longer relevant.
